partcad/src/partcad/context.py

```
@@ -232,6 +232,8 @@ def get_project(self, rel_project_path: str):
                 len_to_skip = 1
             project_path = project_path[len_to_skip:]
 
+            if not self.name in self.projects:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)